tests/behat/features/fga-use-case-c.feature

```diff
@@ -21,6 +21,9 @@ Feature: Use case C: Closely cooperating institutions
                 "select_raa": [
                     "institution-a.example.com",
                     "institution-d.example.com"
+                ],
+                "use_raa": [
+                    "institution-d.example.com"
```

Comment:
Reading the RFC, it seems incorrect to add use-raa for institution d here. Can you explain why this change was introduced? Idem for the use-raa on institution-d in the config below.

Reply:
If an institution is selected because it's allowed in the `select_raa` config it still has to be configured to use the institutions as RA(A) based on the `use_ra(a)` config. This is done to support virtual organizations were an organization is used as an intermediate to manage organizations while not having RA(A)'s itself.

https://www.pivotaltracker.com/story/show/170487212